addons/website/views/snippets.xml

```diff
@@ -1346,7 +1351,7 @@
     <!-- Color | Section -->
     <div id="so_main_colorpicker"
         data-js="colorpicker"
-        data-selector="section, :not(.o_gallery > .container) > .carousel"
```

Comment:
In fact, did you check that the image gallery snippet did not receive a lot of options that do not work with it by changing all these selectors ?

Reply:
The option was only active on carousel that were not part of galleries, as they are handled separately (they also did not suffer the same problem regarding anchors as the carousel was not at top level, and would not be the anchor bearing element). Since I did not add the o_carousel_wrapper class to galleries, this is still the case.

Reply:
I ended up adding `.o_gallery .carousel_item` to data-exclude, which fixed the issue of those options being available on galleries.